app/src/main/java/org/p2p/wallet/newsend/model/SearchResult.kt

```diff
@@ -58,3 +59,8 @@ sealed class SearchResult(open val address: String) : Parcelable {
 
 fun TemporaryAccount.toSearchResult(): SearchResult =
     SearchResult.AddressFound(address)
+
+fun SearchResult.nicknameOrAddress(): String {
```

Comment:
usernameOrAddress, we don't use nickname anywhere in code

Reply:
We do.